datafusion/physical-plan/src/metrics/mod.rs

```diff
@@ -209,6 +209,13 @@ impl MetricsSet {
             .map(|v| v.as_usize())
     }
 
+    /// Convenience: return the total rows of spills, aggregated
+    /// across partitions or `None` if no metric is present
+    pub fn spilled_rows(&self) -> Option<usize> {
+        self.sum(|metric| matches!(metric.value(), MetricValue::SpilledRows(_)))
```

Comment:
A minor one, I wonder if `rows_spilled`, and `bytes_spilled` would be more natural.

Reply:
Thanks for the review. I think tracking legacy naming convention can be more useful but please let me know if there is concern.